client/src/features/projectsV2/fields/ProjectVisibilityFormField.tsx

```diff
@@ -85,14 +70,9 @@ export default function ProjectVisibilityFormField<T extends FieldValues>({
               />
               <Label
                 for="project-visibility-private"
-                className={cx(
-                  "cursor-pointer",
-                  "d-flex",
-                  "align-items-center",
-                  "gap-1"
-                )}
+                className={cx("cursor-pointer")}
               >
-                Private <Lock size={16} />
+                Private <Lock className="bi" />
```

Comment:
```suggestion
                Private
                <Lock className={cx("bi", "ms-1")} />
```

Also, why is the icon after the text? Usual placement is before the text for most icons.

Reply:
Fair point. Swapped and adjusted for both the Private and Public options